### ISSUE TITLE
Operation not found 

### ISSUE THREAD

**Issue by Vikramank** [omimo/HelloTensor#5]
Hi Omimo, 
I ran your scripts, but when I execute this statement, 
`inferenceInterface.fillNodeFloat(INPUT_NODE, INPUT_SIZE, inputFloats);`
it is throwing me the following error. 
`java.lang.IllegalArgumentException: No Operation named [I] in the Graph`

**Comment by omimo** [comment]
Hey, 

Are you using the same TF graph as in the example code? 

The problem seems to be that there is no node in the Tensorflow graph named 'I'. You define it here with name='I':

I = tf.placeholder(tf.float32, shape=[None,3], name='I') # input

**Comment by Vikramank** [comment]
Hi Omimo, 
Yes indeed. But I resolved it. After saving the optimised .pb file, I ran a check to find the nodes. This command apparently removed all the nodes. To be specific, the following lines removed all the nodes present in the graph 
```
f = tf.gfile.FastGFile(output_optimized_graph_name, "wb")
f.write(output_graph_def.SerializeToString())
```
So, I did a small trick.
First I took the frozen graph, optimised it 
`output_graph_def = optimize_for_inference_lib.optimize_for_inference(...)`
then I saved it in .pbtxt format 
`tf.train.write_graph(output_graph_def, '', 'savegraph2.pbtxt')  `
I again used freeze graph option because it will output a graph as .pb file. 
`freeze_graph.freeze_graph()`

This time, the graph def was ready for inference and all the nodes were also present.
